updating/understanding-openshift-updates.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Understanding {product-title} updates
+include::modules/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update a {product-title} cluster with a single click using over the air update.
+Platform administrators are automatically notified when an update is available for their cluster.
+Updates can be applied either from the `oc` command-line interface (CLI) or the web console.
+
+The {product-title} update service builds a graph of upgrade possibilities based on release images in the registry.
+This graph is based on allowable tested update paths from a specific version.
```

Comment:
"allowable" feel controlling.  Cluster administrators are able to make off-graph updates, and sometimes those are still supported via support exceptions.  Can we say `supported, tested update paths` or some such here, or do we need to find a word that doesn't tie so directly to contracts?